app/src/main/java/com/babylon/wallet/android/domain/usecases/VerifyDAppUseCase.kt

```diff
@@ -53,10 +55,11 @@ class VerifyDAppUseCase @Inject constructor(
                 origin = request.metadata.origin,
                 dAppDefinitionAddress = dAppDefinitionAddress
             ).onFailure { error ->
-                error.asRadixWalletException()?.let { radixWalletException ->
-                    val walletErrorType = radixWalletException.toConnectorExtensionError() ?: return@let
-                    respondToIncomingRequestUseCase.respondWithFailure(request, walletErrorType, radixWalletException.getDappMessage())
-                }
+                logNonFatalException(error)
+                val walletErrorType =
+                    error.toConnectorExtensionError() ?: ConnectorExtensionError.UNKNOWN_DAPP_DEFINITION_ADDRESS
+                val message = error.getDappMessage()
+                respondToIncomingRequestUseCase.respondWithFailure(request, walletErrorType, message)
```

Comment:
is it better to rename it to ceError or is it a wallet error?
Please also use the named parameters in `respondWithFailure`.

Reply:
@giannis-rdx I renamed it to `dappWalletInteractionErrorType` to match it's type that is defined in sargon